Tests/AutoPilotTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AutoPilotTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import KeyboardSupport
+
+class AutoPilotTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var viewController: AutoNavigatorViewController! = nil
```

Comment:
Since this is an implicitly unwrapped optional, setting it to `nil` initially seems kind of redundant.